roles/ci-nexus-proxy/tasks/main.yml

```diff
@@ -1,12 +1,11 @@
 ---
 
-- name: Create podman storage config
-  copy: src=etc/containers/storage.conf dest=/etc/containers/storage.conf mode=0644 owner=root group=root
+- name: Copy the installation script for nexus-proxy
+  copy: src=install-nexus3.sh dest=/home/fedora/install-nexus3.sh owner=fedora group=fedora mode=0140
 
-- name: Install podman
-  dnf:
-    name: ['podman']
-    state: present
+- name: Run the installation script for nexus-proxy
+  shell:
+    cmd: ./home/fedora/install-nexus3.sh /home/fedora
```

Comment:
If there is a specific file that indicates Nexus is already set up, you can add something like that: 

https://github.com/hibernate/hibernate-infra/blob/bd7b038356d90d724e793c94bb481db8a204b0f6/roles/websites/tasks/main.yml#L90-L91

Then ansible will skip the script if the file is already present.